tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -330,73 +341,66 @@ def __validate_s2_records_are_related(self):
-
-                    if closure_reason == '01':
-                        num_errors += self.__validate_case_closure_employment(t4, t5s, (
-                            'At least one person on the case must have employment status = 1:Yes in the '
-                            'same RPT_MONTH_YEAR since CLOSURE_REASON = 1:Employment/excess earnings.'
-                        ))
-                    elif closure_reason == '03' and not is_ssp:
-                        num_errors += self.__validate_case_closure_ftl(t4, t5s, (
-                            'At least one person who is head-of-household or spouse of head-of-household '
-                            'on case must have countable months toward time limit >= 60 since '
-                            'CLOSURE_REASON = 03: federal 5 year time limit.'
-                        ))
-                if len(t5s) == 0:
-                    for record, schema in t4s:
-                        self.__generate_and_add_error(
-                            schema,
-                            record,
-                            field='RPT_MONTH_YEAR',
-                            msg=(
-                                f'Every {t4_model_name} record should have at least one corresponding '
-                                f'{t5_model_name} record with the same RPT_MONTH_YEAR and CASE_NUMBER.'
-                            )
+        t4s = self.sorted_cases.get(t4_model, [])
+        t5s = self.sorted_cases.get(t5_model, [])
+
+        if len(t4s) > 0:
+            if len(t4s) > 1:
+                for record, schema in t4s[1:]:
+                    self.__generate_and_add_error(
+                        schema,
+                        record,
+                        field='RPT_MONTH_YEAR',
+                        msg=(
+                            f'There should only be one {t4_model_name} record  '
+                            f'per RPT_MONTH_YEAR and CASE_NUMBER.'
                         )
-                        num_errors += 1
-                else:
-                    # success
-                    pass
+                    )
+                    num_errors += 1
             else:
-                for record, schema in t5s:
+                t4 = t4s[0]
+                t4_record, t4_schema = t4
+                closure_reason = getattr(t4_record, 'CLOSURE_REASON')
+
+                if closure_reason == '01':
+                    num_errors += self.__validate_case_closure_employment(t4, t5s, (
+                        'At least one person on the case must have employment status = 1:Yes in the '
+                        'same RPT_MONTH_YEAR since CLOSURE_REASON = 1:Employment/excess earnings.'
+                    ))
+                elif closure_reason == '03' and not is_ssp:
+                    num_errors += self.__validate_case_closure_ftl(t4, t5s, (
+                        'At least one person who is HoH or spouse of HoH on case must have FTL months >=60.'
```

Comment:
@elipe17 this is the full error message:

`At least one person who is head-of-household or spouse of head-of-household on case must have countable months toward time limit >= 60 since CLOSURE_REASON = 03: federal 5 year time limit.`